cli/apis/auxiliaryFunctions.py

```diff
@@ -18,11 +18,12 @@
 import json
 import os
 from alterations import alteration
+import env
 
 
 def addSensitiveVariable(file, processorName, key, value):
     for processor in file["flowContents"]["processors"]:
-        if processor["name"] == processorName:
+        if processor[env.NAME_TAG] == processorName:
```

Comment:
Isn't this variable the same as the one employed for the alteration?

Reply:
Here you fill the template with a sensitive variable. So, I restored the code because the template structure is independent of our file.